edifact-edi/2.3/modules/ROOT/pages/edifact-edi-connector-reference.adoc

```diff
@@ -0,0 +1,253 @@
+| Initial interchange reference | String |  The initial Interchange Control Reference used for outgoing messages. |  1 |
+| Initial message reference | String |  The initial Message Reference Number used for outgoing messages. |  1 |
+| Request acknowledgments | Boolean |  Request acknowledgments for Send interchanges using Acknowledgment Request field (UNB.9). If true, CONTRL acknowledgments are requested in all sent interchanges. If false, only receipt confirmations are requested. |  false |
+| Test indicator | String |  Test indicator digit to be used on Send interchanges. By default, interchanges are not sent as tests. |  |
+| Use Supplied Values | Boolean | Whether to generate control numbers when writing or use supplied values for control-segment information such as control numbers. The default behavior is to always generate control numbers when writing. This allows you to instead use the values you have chosen. | false |
+| Interchange control number key | String |The key-value pair of the interchange control key and interchange control number.| NONE |
+| Message control number key | String | The key-value pair of the message control key and message control number.| NONE |
+| Truncate values exceeding the length limits | Boolean | Works when enforce length values is `true`. If `true`, values exceeding maximum length are trimmed. If `false`, the values are written as-is. | false |
+|===
+
+
+== Operations
+
+* Read
+* Write
+* Write batch
+
+== Read
+
+`<edifact:read>`
+
+
+Transform an input stream of EDI text into maps and lists structure of EDI data.
+
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | |x
+| Read Content | Binary |  |  `#[payload]` |
+| Target Variable | String |  The name of a variable on which the operation's output is placed. |  |
+| Target Value | String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable. |  `#[payload]` |
+|===
+
+=== Output
+
+[cols="30a,70a"]
+|===
+| Type | Object
+|===
+
+=== For Configurations
+
+* config
+
+=== Throws
+
+* EDIFACT:SCHEMA
+* EDIFACT:WRITE
+* EDIFACT:UNKNOWN
+* EDIFACT:PARSE
+
+== Write
+
+`<edifact:write>`
+
+Transform maps and lists structure of EDI data to EDI text stream.
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | |x
+| Write Content | Object |  |  `#[payload]` |
+| Streaming Strategy | repeatable-in-memory-stream
+repeatable-file-store-stream
+non-repeatable-stream |  Configure to use repeatable streams and their behavior. |  |
+| Target Variable | String |  The name of a variable on which the operation's output is placed. |  |
```

Comment:
Same comment as previous. These updates can be made in a future iteration though. Not important right now.